canonical-web-and-design/ubuntu.com
# Add details and list components

## Done

- Add basic subscription list and subscription details components.
- Make subscriptions list scroll

## QA

- Visit /advantage?test_backend=true.
- You should see cards and details.
- Check that you can scroll the list of cards.


## Issue / Card

Fixes: canonical-web-and-design/commercial-squad#94.
Fixes: canonical-web-and-design/commercial-squad#106.
